ui/app/appointments/controllers/manage/appointmentsCreateController.js

```diff
@@ -19,8 +19,22 @@ angular.module('bahmni.appointments')
             $scope.showStartTimes = [];
             $scope.showEndTimes = [];
 
+            var isProviderCurrentlyAvailableForAppointments = function (selectedProvider, config) {
+                var providers = config.providers;
+                return _.find(providers, function (provider) {
+                    return selectedProvider.uuid === provider.uuid;
+                });
+            };
             var init = function () {
                 wireAutocompleteEvents();
+                var providerCurrentAvailability = false;
+                if (!_.isEmpty(appointmentContext) && !_.isEmpty(appointmentContext.appointment) && !_.isEmpty(appointmentContext.appointment.provider)) {
+                    providerCurrentAvailability = isProviderCurrentlyAvailableForAppointments(appointmentContext.appointment.provider, appointmentCreateConfig);
+                    if (_.isEmpty(providerCurrentAvailability)) {
```

Comment:
is "_.isEmpty()" the right thing to do? shouldn't we instead do?_.isEqual(providerCurrentlyAvailability, false) be more appropriate, if you must use lodash, rather than conventional equality check.